examples/README.md

```diff
@@ -0,0 +1,67 @@
+# Examples
+This directory contains example configs that can be used with PolarRoute.
+
+To run PolarRoute, choose one config out of each directory and execute commands in this order:
+
+1. create_mesh {environment_config}
+2. add_vehicle {vessel_config} {create_mesh_output}
+3. optimise_routes {route_config} {add_vehicle_output} {waypoints}
+4. (optional) plot_mesh {any_output}
```

Comment:
Worth noting somewhere that this requires installing `bas-geoplot`, which is a separate package